source/features/forked-to.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import cache from '../libs/cache';
+import {getRepoURL} from '../libs/utils';
+import {isOwnRepo} from '../libs/page-detect';
+
+const id = 'forked-to';
+
+const currentRepo = getRepoURL();
```

Comment:
I think this function is already cached. You can use it online without further variables.

Reply:
In `utils` its not cached, but then again it's not an intensive function. 👍